.github/workflows/sass.yml

```diff
@@ -0,0 +1,28 @@
+name: Compile sass files
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Why just the master branch?

Reply:
No reason, I was just using the github actions gui and that must be the default. Will fix.